src/cli.ts

```diff
@@ -96,6 +104,15 @@ function humanSizeCaption(n: tree.Node): string {
   return `${n.id || ''} (${size.toFixed(1)}${units[unit]})`;
 }
 
+
+/** Write contents (utf-8 encoded) to a temp file, returning the path to the file. */
+async function writeToTempFile(contents: string): Promise<string> {
+  const randHex = crypto.randomBytes(4).readUInt32LE(0).toString(16);
```

Comment:
My original PR used the `tmp` module for this, but Evan was concerned about adding a new third party dep due to google3 rules. If that's no longer a concern, I'd certainly prefer to use the module.